numba_dpex/core/runtime/_dbg_printer.h

```diff
@@ -13,7 +13,7 @@
 
 /* Debugging facilities - enabled at compile-time */
 /* #undef NDEBUG */
-#if 0
```

Comment:
missed cleanup

Reply:
Sure, I did not clean up yet. Same for some other comments.